tests/humanize_tests/tests.py

```diff
@@ -125,7 +125,7 @@ def test_i18n_intword(self):
             '6000000000000',
         )
         result_list = (
-            '100', '1,0 Million', '1,2 Millionen', '1,3 Millionen',
+            '100', '1,0 Million', '1,2 Million', '1,3 Million',
```

Comment:
@MarkusH, is this change correct for German? If not, I'm not sure how to adapt things. \cc @claudep.